### PR TITLE
RANCHER-1241 create uninstall module pipeline

### PR DESCRIPTION
**Pull request by aatoyan**

### CODE REVIEW

File: pipelines/folioRancher/folioDevTools/moduleDeployment/uninstallModules/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    folioParameters.agent(),
+    folioParameters.refreshParameters()
+    ])
+])
+
+def CLUSTER_URL = params.CLUSTER
+def NAMESPACE_URL = params.NAMESPACE
+def TENANT_URL = params.TENANT_ID
+
+if (params.REFRESH_PARAMETERS) {
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioDevTools/moduleDeployment/uninstallModules/Jenkinsfile:30]: I think this code should be after properties block before other code

File: pipelines/folioRancher/folioDevTools/moduleDeployment/uninstallModules/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    folioParameters.tenantId(''),
+    string(name: 'MODULE_NAME', defaultValue: '', description: '(REQUIRED) Module names separated by comma', trim: true),
+    folioParameters.agent(),
+    folioParameters.refreshParameters()
+    ])
+])
+
+def CLUSTER_URL = params.CLUSTER
+def NAMESPACE_URL = params.NAMESPACE
+def TENANT_URL = params.TENANT_ID
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioDevTools/moduleDeployment/uninstallModules/Jenkinsfile:28]: We have a RancherNamespace object where this info could be stored

File: pipelines/folioRancher/folioDevTools/moduleDeployment/uninstallModules/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+if (!main.isTenantExist(params.TENANT_ID)) {
+  currentBuild.result = 'ABORTED'
+  error('Tenant does not exist, aborting the pipeline')
+}
+
+if (params.MODULE_NAME.trim() == '') {
+    currentBuild.result = 'ABORTED'
+    error('MODULE_NAME parameter cannot be empty, aborting the pipeline')
+}
```
[the remark this answers is not in the document]
Comment (aatoyan) [reply pipelines/folioRancher/folioDevTools/moduleDeployment/uninstallModules/Jenkinsfile:48]: done

File: pipelines/folioRancher/folioDevTools/moduleDeployment/uninstallModules/Jenkinsfile (hunk 1 above)
Comment (aatoyan) [reply pipelines/folioRancher/folioDevTools/moduleDeployment/uninstallModules/Jenkinsfile:30]: done

**Pull request closed without merging**